### PR TITLE
Diffyscan config for shapella upgrade

### PR DESCRIPTION
**Pull request by folkyatina**
Config for verifying contracts code against the etherscan with https://github.com/lidofinance/diffyscan

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "0xC77F8768774E1c9244BEed705C4354f2113CFc09": "DepositSecurityModule",
+    "0x9305c1Dbfe22c12c66339184C0025d7006f0f1cC": "OracleReportSanityChecker",
+    "0xbf05A929c3D7885a6aeAd833a992dA6E5ac23b09": "OracleDaemonConfig",
+    "0x8F73e4C2A6D852bb4ab2A45E6a9CF5715b3228B7": "EIP712StETH",
+    "0xD15a672319Cf0352560eE76d9e89eAB0889046D3": "Burner",
+    "0xD624B08C83bAECF0807Dd2c6880C3154a5F0B288": "HashConsensus",
+    "0x7FaDB6358950c5fAA66Cb5EB8eE5147De3df355a": "HashConsensus"
+  },
+  "network": "mainnet",
+  "github_repo": "https://github.com/lidofinance/lido-dao/tree/35d8c862406f82c64091760aec8feab0e114aac1",
```
Comment (TheDZhon) [inline config.json:25]: I would use the audited commit
https://github.com/lidofinance/lido-dao/commit/e45c4d6fb8120fd29426b8d969c19d8a798ca974

File: config.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "contracts": {
+    "0x852deD011285fe67063a08005c71a85690503Cee": "OssifiableProxy",
```
Comment (TheDZhon) [inline config.json:3]: Can we add the comments what are the proxies here?

**Review by TheDZhon — COMMENTED** [review]
:eyes:

File: config.json (hunk 2 above)
Comment (folkyatina) [reply config.json:3]: There is no comments in json. 🤷

**Review by mymphe — APPROVED** [review]

**Review by kadmil — APPROVED** [review]
LGTM (checked the config in call)

**Pull request merged**